python/cudf/cudf/testing/fuzzer.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import datetime
+import json
+import logging
+import os
+import sys
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+class Fuzzer(object):
+    def __init__(
+        self,
+        target,
+        data_handler_class,
+        dirs=None,
+        crash_reports_dir=None,
+        regression=False,
+        max_rows_size=4096,
```

Comment:
This default is used now?

Reply:
The default in `main.py` is the one being used. Changed this one to be uniform too..